contrib/scripts/check-helm-quick-install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+MAKE=${MAKE:-make}
+HELM_BIN=${HELM_BIN:-helm}
+HELM_VERSION=${HELM_VERSION:-3.1.1}
```

Comment:
I'd suggest to define the default helm version in a more central place (e.g. `Makefile.def`) so it can potentially be reused for other checks, e.g. regarding #10418

Reply:
Yeah, I think it makes sense. I have updated the pull request. Can you take a look?